csharp/ql/src/Security Features/CWE-016/ASPNetMaxRequestLength.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Large maxRequestLength value.
+ * @description maxRequestLength should be set as minimum as possible.
+ * @kind problem
+ * @problem.severity warning
+ * @precision very-high
```

Comment:
Please remove the precision tag (we will make sure to readd it in a follow-up PR.)